### DISCUSSION TITLE
查询某个Key用的什么方法，速度是否比较快

### DISCUSSION THREAD

**Discussion by justinleeyang** [armink/FlashDB#96] [General]
大家好，

我想了解FlashDB如果要查询某个key，用的什么方法，如果再STM32F405的平台，查询的速度怎么样？

**Comment by armink** [comment]
可以适配到 F405 上，跑下 bench 就可以了

**Reply by justinleeyang** [reply]
你好，请问如果我有100个key，这时候来一个已经有的key，我想覆盖掉之前的数据，请问目前是怎么查询到这个Key的，是轮询吗？还是其他方法？ 主要我的Key可能比较多，感觉轮询时间会太久了

**Reply by armink** [reply]
如果 kv_set 是一个已有的 key ，默认就会覆盖，不需要用户再轮训检查
